libs/sdk-ui-charts/src/highcharts/constants/label.ts

```diff
@@ -1,14 +1,14 @@
-// (C) 2019-2020 GoodData Corporation
+// (C) 2019-2021 GoodData Corporation
 import Highcharts from "../lib";
 import { VisualizationTypes } from "@gooddata/sdk-ui";
 
 export const WHITE_LABEL: Highcharts.CSSObject = {
-    color: "#ffffff",
-    textShadow: "0 0 1px #000000",
+    color: "var(--gd-palette-complementary-9, #ffffff)",
+    textShadow: "0 0 1px var(--gd-shadow-color, #000000)",
 };
 
 export const BLACK_LABEL: Highcharts.CSSObject = {
-    color: "#000000",
+    color: "var(--gd-palette-complementary-9, #000000)",
```

Comment:
Are these complementary indices correct? It's strange that both black and white map to 9...